### DISCUSSION TITLE
Improving Ansel Culling Workflow

### DISCUSSION THREAD

**Discussion by arky** [aurelienpierreeng/ansel#100] [General]
I tend spend considerable amount of time with lighttable culling. Usually the processing is much more time intensive than post-processing. 

So I think it would be valuable if we could make small paper cut improvements to the process.  

Few things that come to my mind are improving are: showing panel overlays (RAW, Reject, Star rating, color icons) by default in lighttable, adding shortcuts for reject, star rating, colors, delete (from disk). 

Any thoughts ?

**Comment by marcsitkin** [comment]
I believe many of those features are there.
![image](https://user-images.githubusercontent.com/58574122/220765535-db780a64-771d-447b-9fb3-d1f7d9c37e87.png)
You can choose overlay visibility by clicking on the star
![image](https://user-images.githubusercontent.com/58574122/220765737-ee6d44fe-da20-490d-b65c-01c8e51ce771.png)

Star ratings can be added to selected images using numbers 1-5, R for reject. Fn and function key to assign a color label.

H key shows shortcuts.

Are you very new to this software? The darktable manual might be helpful to you to read.

Apologies if I misunderstood what you are looking for.

**Comment by arky** [comment]
I am aware that these features exist and are well documented.  

The purpose is of this post is to explore minor improvements (papercuts) to make the software more accessible for new users. 

For instance, if we could enable lighttable overlays by default,  it could make things much simpler.  Unless I am mistaken, the default overlay setting is none.

**Comment by arky** [comment]
Perhaps providing examples would help the discussion. 

In this test case, I am trying to remove discard an image from the disk. You can select an image and press `delete` key and it will `Remove` the file from Ansel's library, but there is no shortcut for `delete (trash)`.

Users could open preferences and try to set a custom shortcut. However there is no 'delete (trash)`  in `Act on selection` ? 

![image](https://user-images.githubusercontent.com/808895/220827713-1dc91270-4a2b-442f-9ece-85c2e6fb8311.png)

**Comment by lu-k** [comment]
The `delete` you have highlighted actually is the `delete (trash)` action. I use it with a custom shortcut. But yes the shortcut menu is hard to navigate especially to new users and I try to avoid it.

**Reply by arky** [reply]
How did you manage to custom shortcut to `delete (trash)` I never managed to find it in the shortcut preferences? Am using Ansel-0.0.0+107~gd9e1b5ae5-x86_64 on gnu/Linux.

**Reply by lu-k** [reply]
I'm afraid I don't understand. You are showing the right item in the screenshot above? There are `remove` (from Library) and `delete` (from disk, using trash) to be distinguished.

Sometimes I use that "magic keyboard shortcut tool" if I can't be bothered scrolling through the millions of options. In the case of `delete` it will bring you to the same window you have in your screenshot. Just make sure you exit the "magic keyboard shortcut tool" with right-click... 
![Screenshot from 2023-02-28 15-54-58](https://user-images.githubusercontent.com/64734384/221890870-00ada62c-d561-4ab3-a6b9-8c6f97740aaa.png)

**Reply by arky** [reply]
Sweet! I didn't know this `magic keyboard shortcut tool` trick. Thank you. I have been searching and scrolling through the shortcuts preferences window.

**Reply by lu-k** [reply]
It's officially called [visual shortcut mapping ](https://docs.darktable.org/usermanual/4.0/en/preferences-settings/shortcuts/#visual-shortcut-mapping). I have linked to the darktable manual, because the Ansel manual doesn't show the icons atm.

**Reply by arky** [reply]
Thanks!
